src/Microsoft.AspNet.OData.Shared/DefaultPatchHandler.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.OData
+{    
+    internal class DefaultPatchHandler<TStructuralType> : PatchMethodHandler<TStructuralType> where TStructuralType :class
```

Comment:
>DefaultPatchHandler [](http://example.com/codeflow?start=19&length=19)

Add comments describing this class, its behavior, and where/when it is appropriate to use.